packages/accordion/README.md

```diff
@@ -1,13 +1,7 @@
 # [`@fremtind/jkl-accordion`](https://fremtind.github.io/jokul/components/accordion/)
 
-## Beskrivelse
+# Trekkspillister
+Med trekkspillister kan vi presentere mye innhold på liten plass. En trekkspilliste har én eller flere rader, som brukeren kan trykek på for å vise eller skjule informasjon.
 
-Trekkspillister gjør at vi kan gi brukeren mer innhold på liten plass, ved å vise informasjonen gradvis. De lar brukeren selv velge å vise eller skjule mer innnhold.
-
-En trekkspilliste kan bestå av en eller flere rader. Brukeren kan klikke eller trykke på enten teksten eller vinkeltegnet (chevronen) for å vise eller skjule informasjonen. Vi legger ikke trekkspillister inn i andre trekkspillister (nesting). 
-
-Vurder hvor viktig infomasjonen er før du velger om du skal bruke en trekkspilliste. Viktig informasjon bør helst ligge åpent tilgjengelig. Da velger du enten å ikke ha infoen i trekkspilliste, eller du setter trekkspillisten som åpen, som standard.
-
-## Eksempler
-- Liste med spørsmål og svar (for eksempel Vanlige spørsmål (FAQ)
-- Tilleggsinformasjon som ikke er viktig for de fleste (for eksempel mer informasjon om Trafikkforsikringsavgiften)
+## Bruk
+Viktig informasjon bør alltid ligge synlig, ikke skjult i en trekkspilliste. Trekkspillister passer godt til tilleggsinformasjon som kan være viktig for brukerne, men som de selv tar valget om å se på. Vinkeltegnet indikerer hva som skjer når brukerne klikker på det: da åpner og lukker listen seg. Trekkspillister må ha en god overskrift, som forteller hva som kommer. Teksten på hver rad i listen må være tydelig og beskrive innholdet for brukeren.
```

Comment:
Disse setningene slår vel hverandre ihjel? Hva med å `Trekkspillister passer godt til tilleggsinformasjon som kan være nyttig for brukerne, men som de selv tar valget om å se på`

Reply:
Omformulert og korrkturlest